tests/framework/extensions/provisioning/delete.go

```diff
@@ -0,0 +1,26 @@
+package provisioning
```

Comment:
I honestly don't think we need this at all. We only allow testing.T if it's a verification look at `extensions/provisioning/verify.go`